.github/CONTRIBUTING.md

```diff
@@ -30,9 +30,21 @@ If you write and contribute a full topic, we will add your name (or your company
 1. Create a new branch on your fork. Use a name that best describes the work, or references a GitHub issue number.
 1. Edit or create markdown (`.md`) files in your branch.
 1. When ready, push your branch to your fork.
-1. Create a PR to the [magento/devdocs repo](https://github.com/magento/devdocs). Fill out as much info as possible and link any GitHub issues.
 
-   In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, like `2.3.1-integration`.
+### Create a pull request
+
+Create a PR to the [magento/devdocs repo](https://github.com/magento/devdocs). Fill out as much info as possible and link any GitHub issues.
```

Comment:
Avoid abbreviations.
Change _repo_ to _repository_, _info_ to _information_.

Reply:
Done.